SampleTest/MainPage.xaml

```diff
@@ -280,8 +283,8 @@
                             </StackPanel>
                             <wgt:GraphPresenter Grid.Row="1"
                                                 IsCollection="True"
-                                                RequestBuilder="{x:Bind local:MainPage.GetTeamsChannelMessagesBuilder('02bd9fd6-8f93-4758-87c3-1fb73740a315', '19:d0bba23c2fc8413991125a43a54cc30e@thread.skype'), Mode=OneWay}"
```

Comment:
Ah, so the late bind provider causes the Provider Instance to be null in the function, eh?

Hmm, will have to think about how we can have dynamic binding here for other scenarios. The Teams scenario is a bit odd as an example for this one. May need another helper or something... Eventually we'll want to be able to have folks have some way of playing with this live in the Sample App, but we can worry about that later.

Reply:
Indeed, the new setup broke that binding pattern. I'm sure we can come up with a helper for this that is aware of GlobalProvider changes.